README.md

```diff
@@ -5,9 +5,9 @@ Code styles, and tools for different languages
 
 - Clients should send the next headers on every request to our api:
 
-  * `bv-app-version`: `platform/typeApp/version/build` (***each value separated by a `/`***) _where_:
+  * `bv-app-version`: `platform/typeApp/version/build` (**each value separated by `/`**) where:
 
-    - `platform`***(mandatory)*** : platform who make the request in lower case (ej: ios, android)
-    - `typeApp`***(mandatory)*** : app profile who make the request in lower case (ej: customer, driver, admin) if there are only one app could be mobile for mobile apps
-    - `version`***(mandatory)*** : current app version following semver standard (x.y.z)
-    - `build`***(optional)*** : current build version, could be number or letters
+    - `platform`: platform who make the request in lower case (ej: ios, android)
```

Comment:
@earayo I prefer say just whats optional like swagger models :P, but the fix is not for that, is to fix a markdown issue